pkg/metadata/inventories/inventories.go

```diff
@@ -460,7 +461,8 @@ func initializeConfig(cfg config.Config) {
 	SetAgentMetadata(AgentCWSNetworkEnabled, config.SystemProbe.GetBool("event_monitoring_config.network.enabled"))
 	SetAgentMetadata(AgentCWSSecurityProfilesEnabled, config.SystemProbe.GetBool("runtime_security_config.activity_dump.enabled"))
 	SetAgentMetadata(AgentCWSRemoteConfigEnabled, config.SystemProbe.GetBool("runtime_security_config.remote_configuration.enabled"))
-	SetAgentMetadata(AgentProcessEnabled, config.Datadog.GetBool("process_config.process_collection.enabled"))
+	SetAgentMetadata(AgentProcessEnabled, cfg.GetBool("process_config.process_collection.enabled"))
```

Comment:
Could you update the entire function to use `cfg` instead of `config.Datadg` (ie: line 459, 465 and 465). Thanks!

Reply:
Fixed ! I also added a unit test.